### PR TITLE
A little late with this (turkies)

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
turkies for thanksgiving

## Why It's Good For The Game
turk
who would say no to this fella

## Changelog
:cl: Mark Suckerberg and Thatguythere03
add: turkies
soundadd: turky noises
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
Do they lay chicken eggs, which hatch chickens...?

**Comment by qwertyquerty** [comment]
Don't think about it too much

**Comment by ike709** [comment]
Just set `egg_type` to null and they won't lay eggs.

**Comment by Moccha-Bee** [comment]
Cute, might make dinner with it later

**Comment by MarkSuckerberg** [comment]
aww fine

**Comment by MarkSuckerberg** [comment]
travis wants them laying eggs, it's irrefutable

**Comment by ike709** [comment]
@MarkSuckerberg `null` has to be lowercase.

File: code/modules/mob/living/simple_animal/friendly/farm_animals.dm [hunk 1]
```diff
@@ -324,6 +324,27 @@
...
+	icon_living = "turkey_plain"
+	icon_dead = "turkey_plain_dead"
+	speak = list("Gobble!","GOBBLE GOBBLE GOBBLE!","Cluck.")
+	speak_emote = list("clucks","gobbles")
+	emote_hear = list("gobbles.")
+	emote_see = list("pecks at the ground.","flaps its wings viciously.")
+	density = FALSE
+	health = 15
+	maxHealth = 15
+	egg_type = NULL
```
Comment (ike709) [inline code/modules/mob/living/simple_animal/friendly/farm_animals.dm]:
```suggestion
	egg_type = null
```
Just commit the code suggestion.

**Comment by qwertyquerty** [comment]
@marksuckerberg do the thing it's like one button press

**Pull request merged**